test/lib/Elastica/Test/Query/TermsTest.php

```diff
@@ -36,14 +36,38 @@ public function testFilteredSearch()
         $this->assertEquals(3, $resultSet->count());
     }
 
+    public function provideMinimumArguments()
+    {
+        return [
+            [
+                3
+            ],
+            [
+                -2
+            ],
+            [
+                '75%'
+            ],
+            [
+                '-25%'
+            ],
+            [
+                '3<90%'
+            ],
+            [
+                '2<-25% 9<-3'
+            ]
+        ];
+    }
+
     /**
      * @group unit
+     * @dataProvider provideMinimumArguments
```

Comment:
It would be nice to also have this for an integration test to make sure it also works with the current elasticsearch version. but we can add this in an other PR.